Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -0,0 +1,5 @@
+class _dafny:
+    def print(self, toPrintVal):
+        if type(toPrintVal)== bool:
+            print("true" if toPrintVal else "false")
+helpers = _dafny()
```

Comment:
Can you remove the "self" parameter from `_print` and get rid of `helpers` so that you can simply call statically `_dafny.print(value)` in the emitted code?

Reply:
Also, make sure you add an `else:` to cover all cases (and fail accordingly)